azure/src/test/java/com/microsoft/azure/management/TestContainerInstanceWithPublicIpAddress.java

```diff
@@ -83,7 +94,12 @@ public ContainerGroup createResource(ContainerGroups containerGroups) throws Exc
         Assert.assertEquals(1, nginxContainer.environmentVariables().size());
         Assert.assertTrue(containerGroup.tags().containsKey("tag1"));
         Assert.assertEquals(ContainerGroupRestartPolicy.NEVER, containerGroup.restartPolicy());
+        Assert.assertTrue(containerGroup.isManagedServiceIdentityEnabled());
```

Comment:
can you also add a check here to validate that there are no userAssignedIdentities on the object?

Also, I didn't see either of the tests (this or the one above test userassigned identities)

Can you add tests that : 
 * Assign 2 User Assigned Identities
 * Assign a system assigned and a user assigned identity